### DISCUSSION TITLE
Problem with font "Plantin Std", unnecessary wordwrap in some cases

### DISCUSSION THREAD

**Discussion by IshmaZX82** [SixLabors/Fonts#378] [General] [answered]
**Description**
A strange bug occurs when working with the "Plantin Std" font.
If the first line measured contains a line feed, then in subsequent lines the space is counted as a line feed.

Below is the code that reproduces the problem.
The size of text1 is always measured as { width=1729, height=4096 }.
If text2 is measured before text1, it will be the correct size { width=4027, height=2049 }, without line breaks.
But if text2 is measured after text1 (as in the code below), then the size will be { width=1729, height=4096 }, as in text1.

**Steps to Reproduce**
```
SixLabors.Fonts.FontCollection fontCollection = new SixLabors.Fonts.FontCollection();
fontCollection.Add(@"PlantinStdRegular.otf");
var fontFamily = fontCollection.Get("Plantin Std");
var sixFont = fontFamily.CreateFont(2048f, SixLabors.Fonts.FontStyle.Bold);
var textOptions = new SixLabors.Fonts.TextOptions(sixFont);
textOptions.WrappingLength = float.MaxValue;

var text1 = "D\r\nD"; 
var rect1 = SixLabors.Fonts.TextMeasurer.MeasureAdvance(text1, textOptions);  // always w=1729, h=4096 

var text2 = "D D";
var rect2 = SixLabors.Fonts.TextMeasurer.MeasureAdvance(text2, textOptions);  // (w=1729, h=4096), but must be (w=4027, h=2049)
```

Font:
[PlantinStdRegular.zip](https://github.com/SixLabors/Fonts/files/14182884/PlantinStdRegular.zip)

**System Configuration**
Win10 Pro
Net6 project
SixLabors.Fonts v2.0.1 package

**Comment by JimBobSquarePants** [comment, marked as answer]
Thanks for providing the detail to easily replicate the issue. Very much appreciated!

It's caused by a cache collision in the glyph metrics cache. https://github.com/SixLabors/Fonts/pull/379

I've merged a fix and will look to do an official release soon. In the interim you can pull the nightlies.

**Reply by JimBobSquarePants** [reply]
Fix is released.
